repose-aggregator/components/filters/openstack-identity-v3-filter/src/main/resources/META-INF/schema/config/openstack-identity-v3.xsd

```diff
@@ -364,7 +364,7 @@
             <xs:annotation>
                 <xs:documentation>
                     <html:p>
-                        Cache timeout variance (in milliseconds) for cached data. A random value
+                        Cache timeout variance (in seconds) for cached data. A random value
```

Comment:
It should be in the upgrade guide. We made the switch from milliseconds to seconds to be consistent with the `keystone-v2` filter, and because millisecond resolution does not make much sense in the context of Identity object caching.

On it.